index.js

```diff
@@ -34,6 +34,32 @@ Retryme.prototype.attempt = function (action, fn) {
   }));
 };
 
+Retryme.prototype.async = function (asyncfn) {
+  return new Promise((fulfill, reject) => {
+    debug('Start retries with awaiting async attempt function');
+
+    this.attempt(
+      async next => {
```

Comment:
similar question here, is it okay to force a min-node version or babel+regenerator functions on client-side consumers here for async..await in the library?

Reply:
we could convert this to a `new Promise` which would reduce support worries to just IE11 perhaps?